src/webportal/src/app/job-submission/utils/team-storage-client.js

```diff
@@ -0,0 +1,44 @@
+export class TeamStorageClient {
+  constructor(api, user, token) {
+    this.userInfoUrl = `${api}/api/v2/user/${user}`;
+    this.storageConfigUrl = `${api}/api/v1/kubernetes/api/v1/namespaces/pai-storage/secrets/storage-config`;
+    this.storageServerUrl = `${api}/api/v1/kubernetes/api/v1/namespaces/pai-storage/secrets/storage-server`;
+    this.token = token;
+  }
+
+  async fetchUserGroup() {
```

Comment:
May be these functions can move to utils/conn.js

Reply:
that makes sense